code/modules/power/powernet.dm

```diff
@@ -334,6 +334,9 @@ var/global/powernets_broke = 0
 			if(G.siemens_coefficient == 0)									// to avoid spamming with insulated glvoes on
 				return 0
 
+			if(G.name == "insulted gloves")
```

Comment:
My brother in christ, please just do a typecheck here instead of checking for the gloves' name.

Reply:
i do a name check cause otherwise the budget insulos don't turn insulted on april fools

Reply:
In that case, just transform the budget insuls into insulted gloves when equipped during April Fools.

Reply:
i wanted them to spawn transformed into the insulted ones instead (and I don't feel like remapping), that was the simplest option

Reply:
In that case just add an April fools check in the budget gloves New() proc which spawns insulted gloves and qdels the budget gloves

Reply:
guess that works, lemme see what I can do

Reply:
did the thing